Zend/zend_objects.c

```diff
@@ -247,7 +247,7 @@ ZEND_API void ZEND_FASTCALL zend_objects_clone_members(zend_object *new_object,
 
 	if (old_object->ce->clone) {
 		GC_ADDREF(new_object);
-		zend_call_known_instance_method_with_0_params(new_object->ce->clone, new_object, NULL);
+		zend_call_known_instance_method_with_1_params(new_object->ce->clone, new_object, NULL, old_object);
```

Comment:
what is the right way to pass `old_object` as 1st param?

Reply:
```
zval param;
ZVAL_OBJ(&param, old_object);
```

Reply:
thanks